Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -58,6 +58,7 @@
 static const float s_kCGGradientOffsetPoint = 1E-45;
 
 enum _CGCoordinateMode : unsigned int { _kCGCoordinateModeDeviceSpace = 0, _kCGCoordinateModeUserSpace };
+enum _CGAntialiasToggle : unsigned int { _kCGToggleDefault = 2, _kCGToggleOff = 0, _kCGToggleOn = 1 };
```

Comment:
This isn't antialiasing-specific; perhaps something like _CGTrinary would be better.